app/models/resource.rb

```diff
@@ -11,7 +11,7 @@ class Resource < ActiveRecord::Base
 
   has_and_belongs_to_many :categories
   has_and_belongs_to_many :keywords
-  has_one :address, dependent: :destroy
+  has_many :address, dependent: :destroy
```

Comment:
should probably update this to be plural

Reply:
I believe this has to be updated via a migration (`rails generate migration ...`) rather than manually changing the file

Reply:
@Maxastuart researched this together and confirmed it does not need a migration.